InfinispanAPIObject.adoc

```diff
@@ -0,0 +1,36 @@
+== Infinispan API
+
+=== Motivation
+
+Being able to offer a simple API to create clustered caches and improve API usability
+
+
+== Infinispan
+
+This factory will help to create a cluster of Infinispan.
+
+Whenever a new member is added on the VM, the implicit configuration will handle the visibility between members.
+
+```java
+
+public final class Infinispan {
+
+    private Infinispan() {}
+
+    //Constructs and starts a new instance of the CacheManager, using the system defaults.
+    public static EmbeddedCacheManager createClusteredCache() {}
```

Comment:
Assuming that the InfinispanAPI will be an access point for both caches, locks etc., I wonder if we want a static method `createCacheManager()`. I am wondering if we don't rather want Infinispan object bound to one cache manager, and relaying the other builiding blocks to the same CM:
```java
class Infinispan {
   static Infinispan newInstance();
   EmbeddedCacheManager caches();
   ClusteredLockManager locks();
   ...
}
```